src/app/component/Slider/Slider.style.scss

```diff
@@ -47,6 +56,15 @@
 
         &_isVertical {
             flex-direction: column;
+            width: 100%;
+            max-height: var(--slide-height);
```

Comment:
How is this related to compare? :D

Reply:
I've merged a branch 3.2.5-rc in order to update product card design and put a compare button there.
This comes from the 0ddade4